### ISSUE TITLE
Linear Regression issue

### ISSUE THREAD

**Issue by LabiKSV** [LabiKSV/intro-to-ai-farhan-labi#2]
The linear regression code is showing an issue (as shown below)

![int64 prob](https://user-images.githubusercontent.com/72261917/145877317-80149e72-c072-41f7-9fab-9b052d7496fb.png)

When tracking back, the issue seems to have something to do with the indexing, right at the point where the code counts the missing values. 

From my knowledge it seems like the code is trying to find the index in train_index, under the column string judging by the following bit of code under indexing.py:

![indexing issue](https://user-images.githubusercontent.com/72261917/145883226-dc2e7dc4-0d11-4dcc-b2ef-d3c5c0d85898.png)

Simply put, the code is attempting to find the values of 'train_index' in 'columns' but columns is literally a string containing "Growth Factor of New Deaths"

**Comment by LabiKSV** [comment]
The problem has come down to the sample sizes of train_index and test_index being different. So I'll have to reshape it or not use it, depending on the direction of which e are trying to go at
![inconsistentSample](https://user-images.githubusercontent.com/72261917/146831745-9f1b8cbf-0ca5-4e6c-9014-d7dc59b969f9.png)
